### PR TITLE
Cross Development Build

### PR DESCRIPTION
**Pull request by victorprocure**
Made a couple changes to allow easier building in Powershell/Windows Command Line

### CODE REVIEW

**Comment by TheRobBrennan** [comment]
Hi @victorprocure ! Thank you for creating this PR. I'm not totally sure if I'd want to see this in the `master` branch, but I do think there is definite value in having this in the repo. 

I've created a `nextjs-for-windows-powershell-development` branch to consider this a bit more. Will comment and share some thoughts in a bit!

File: package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
     "test:watch": "./node_modules/.bin/jest --watch",
     "test:coverage": "npm test -- --coverage --no-cache",
     "test:coverage:view": "npm test -- --coverage --no-cache && open coverage/lcov-report/index.html",
-    "dev": "npm run build && NODE_ENV=development node --inspect server/server.js -p $PORT",
-    "prebuild": "rm -rf .next/",
+    "dev": "npm run build && cross-env NODE_ENV=development node --inspect server/server.js -p $PORT",
+    "prebuild": "rimraf .next/",
     "build": "next build",
-    "postbuild": "cp -r static .next/",
+    "postbuild": "cpy static .next/",
```
Comment (TheRobBrennan) [inline package.json]: I'm curious...With the `copy-cli` dev dependency, should this be `copy` instead of `cpy`?

File: package.json [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
     "prestart": "npm run build",
-    "start": "NODE_ENV=production node server/server.js -p $PORT",
+    "start": "cross-env NODE_ENV=production node server/server.js -p $PORT",
```
Comment (TheRobBrennan) [inline package.json]: It's great to see `cross-env` moved to its proper location (`devDependecies`) and actually used. I think I must've used this in a previous demo; and curiously did not use it for some reason. Nice touch.

**Comment by TheRobBrennan** [comment]
@victorprocure I've updated the README so that folks using the main `master` branch - or any of the other supported branches - are aware of the new branch specifically to make life easier for Windows developers:

**Comment by RaffysWeb** [comment]
I've tried to use this repo with docker on windows but docker it's not tracking the changes made to the pages/components.
They only take place if I stop the container and start it again.

**Comment by TheRobBrennan** [comment]
Good evening @RaffysWeb ! First, I'm stoked to see you are using this project. Thank you! 

NextJS has since gone through some major upgrades - up to version 8.x.x as of today - but this project can certainly serve as an on-ramp to experimenting with it.

The reason why the Docker container is not reflecting changes until it's stopped and started again is because of how the `docker-compose.yml` file is set up - https://github.com/TheRobBrennan/demo-nextjs-v6/blob/master/docker-compose.yml

By default, I have local mapping turned off because I was using this project as a starting point for deploying quick and easy NextJS apps. For that to work, the containers needed to be self-contained.

The good news is modifying that file is super easy. If you go and change:
```
    # If you want local changes to update the Docker container, you can pass through project files like this:
    volumes:
      - ./:/srv/www
    # volumes:
    #   - ./layout:/srv/www/layout
    #   - ./pages:/srv/www/pages
    #   - ./server:/srv/www/server
    #   - ./static:/srv/www/static
```

to 

```
    # If you want local changes to update the Docker container, you can pass through project files like this:
    volumes:
      - ./:/srv/www
      - ./layout:/srv/www/layout
      - ./pages:/srv/www/pages
      - ./server:/srv/www/server
      - ./static:/srv/www/static
```

and then spin everything back up again, you should be in great shape.

Cheers,

Rob

**Pull request merged**